### PR TITLE
cr: wait the criu process properly

### PR DESCRIPTION
**Pull request by avagin**

### CODE REVIEW

File: container_linux.go [hunk 1]
```diff
@@ -332,18 +332,18 @@ func (c *linuxContainer) Restore(process *Process) error {
...
-	}
-*/
+	/*
+		// remount root for restore
+		if err := syscall.Mount(c.config.Rootfs, c.config.Rootfs, "bind", syscall.MS_BIND|syscall.MS_REC, ""); err != nil {
+			return err
+		}
 
-	defer syscall.Unmount(c.config.Rootfs, syscall.MNT_DETACH)
+		defer syscall.Unmount(c.config.Rootfs, syscall.MNT_DETACH)
```
Comment (vmarmol) [inline container_linux.go]: why not just remove the line?

Comment (avagin) [inline container_linux.go]: @vmarmol because this code is related from code which is commented out

**Comment by vmarmol** [comment]
LGTM, just sad about the dead code but nothing major.

**Comment by dqminh** [comment]
LGTM

**Pull request merged**